Linking error with Xcode 10.2 toolchain

I'm unable to build SwagGen with the Xcode 10.2 toolchain. There is a linking issue. Reproduced on another computer, so it's not limited to my setup.

```sh
$make install

swift build --disable-sandbox -c release -Xswiftc -static-stdlib
ld: warning: directory not found for option '-L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift_static/macosx'
ld: warning: Could not find auto-linked library 'swiftFoundation'
ld: warning: Could not find auto-linked library 'swiftDarwin'
ld: warning: Could not find auto-linked library 'swiftCoreFoundation'
ld: warning: Could not find auto-linked library 'swiftIOKit'
ld: warning: Could not find auto-linked library 'swiftCore'
ld: warning: Could not find auto-linked library 'swiftCoreGraphics'
ld: warning: Could not find auto-linked library 'swiftObjectiveC'
ld: warning: Could not find auto-linked library 'swiftDispatch'
Undefined symbols for architecture x86_64:
  "_$s10Foundation10NSNotFoundSivg", referenced from:
      _$s10Foundation4DateV4YamsE9construct4fromACSgAD4NodeO6ScalarV_tFZ in Constructor.swift.o
      _$s4Yams8ResolverC10resolveTag2ofAA0D0C4NameVAA4NodeO_tF in Resolver.swift.o
      _$s4Yams3TagC8resolved4withACx_tAA0B10ResolvableRzlFAA4NodeO6ScalarV_Tg5Tf4xn_g in Tag.swift.o
  "_$s10Foundation12CharacterSetV11whitespacesACvgZ", referenced from:
      _$s15StencilSwiftKit7FiltersO7StringsO14removeNewlines_9argumentsypSgAH_SayAHGtKFZ in Filters+Strings.swift.o
      _$sSlsE4drop5while11SubSequenceQzSb7ElementQzKXE_tKFSS17UnicodeScalarViewV_Tg504$ss7f2O6g160VSbs5Error_pIgydzo_ADSbsAE_pIegndzo_TR0116$s15StencilSwiftKit7FiltersO7StringsO24removeLeadingWhitespaces33_BA19D16811ACAA4D0FD5CFE410EC1D8FLL4fromS2S_tFZSbs7a2J16B6VXEfU_Tf3npf_nTf1cn_nTf4gX_n in Filters+Strings.swift.o
      _$s7SwagGen15GenerateCommandC7executeyyKF in main.swift.o
  "_$s10Foundation12CharacterSetV12charactersInACSSh_tcfC", referenced from:
      _globalinit_33_2FABBAEE59BCC804A805D0C499A932AF_func15 in Representer.swift.o
  "_$s10Foundation12CharacterSetV13alphanumericsACvgZ", referenced from:
      _globalinit_33_CB635B249A4735A920030995EC03E33E_func3 in Parameters.swift.o
      _$s10SwagGenKit13CodeFormatterC12escapeStringyS2SFTf4nd_nTf4x_n in CodeFormatter.swift.o
 
...

_$s13JSONUtilities7KeyPathOSayypGypSDySSypGs5Error_pIegggnrzo_AcDypAEsAF_pIegggnozo_TRTA in Dictionary+JSONKey.swift.o
      _$s13JSONUtilities7KeyPathOSayypGypSDySSypGs5Error_pIgggnozo_AcDypAEsAF_pIegggnrzo_TRTA in Dictionary+JSONKey.swift.o
NUtilitiesAA7JSONKeyRzrlE4json9atKeyPath20invalidItemBehaviourSayqd__GAA0eF0O_AA07InvalidhI0Oyqd__GtKSYRd__AA11JSONRawType8RawValueRpd__lFqd__AH_SayypGyptKXEfU_ in Dictionary+JSONKey.swift.o
      ...
ld: symbol(s) not found for architecture x86_64
<unknown>:0: error: link command failed with exit code 1 (use -v to see invocation)

make: *** [build] Error 1
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -14,6 +14,9 @@ jobs:
       - run:
           name: Build Spec Fixtures
           command: ./build_specs.sh
+      - run:
+          name: Build Release
+          command: make build
   macOS_swift_5.0:
     macos:
       xcode: "10.2.0"
@@ -28,6 +31,9 @@ jobs:
       - run:
           name: Build Spec Fixtures
           command: ./build_specs.sh
+      - run:
+          name: Build Release
+          command: make build
 workflows:
   version: 2
   workflow:

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -14,8 +14,23 @@ jobs:
       - run:
           name: Build Spec Fixtures
           command: ./build_specs.sh
+  macOS_swift_5.0:
+    macos:
+      xcode: "10.2.0"
+    steps:
+      - checkout
+      - run:
+          name: Build
+          command: swift build
+      - run:
+          name: Test
+          command: swift test
+      - run:
+          name: Build Spec Fixtures
+          command: ./build_specs.sh
 workflows:
   version: 2
   workflow:
     jobs:
       - macOS_swift_4.2
+      - macOS_swift_5.0

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -1,22 +1,5 @@
 version: 2
 jobs:
-  macOS_swift_4.2:
-    macos:
-      xcode: "10.0.0"
-    steps:
-      - checkout
-      - run:
-          name: Build
-          command: swift build
-      - run:
-          name: Test
-          command: swift test
-      - run:
-          name: Build Spec Fixtures
-          command: ./build_specs.sh
-      - run:
-          name: Build Release
-          command: make build
   macOS_swift_5.0:
     macos:
       xcode: "10.2.0"
@@ -38,5 +21,4 @@ workflows:
   version: 2
   workflow:
     jobs:
-      - macOS_swift_4.2
       - macOS_swift_5.0

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,9 @@
 # Change Log
 ## Next Version
 
+### Changed:
+- Updated to Swift 5 and dropped Swift 4.2 #139
+
 ## 4.0.0
 
 ### Added

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,8 +1,16 @@
 # Change Log
 ## Next Version
 
+## 4.1.0
+
+### Fixed
+- Fixed installing in Swift 5 #139
+- Fixed Swift template building in Swift 5 by Alamofire #139
+
 ### Changed:
-- Updated to Swift 5 and dropped Swift 4.2 #139
+- Updated codebase to Swift 5 and dropped Swift 4.2 #139
+
+[Commits](https://github.com/yonaskolb/SwagGen/compare/4.0.0...4.1.0)
 
 ## 4.0.0
 

--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@ RELEASE_TAR = $(REPO)/archive/$(VERSION).tar.gz
 SHA = $(shell curl -L -s $(RELEASE_TAR) | shasum -a 256 | sed 's/ .*//')
 
 build:
-	swift build --disable-sandbox -c release -Xswiftc -static-stdlib
+	swift build --disable-sandbox -c release
 
 install: build
 	mkdir -p $(PREFIX)/bin

--- a/Package.swift
+++ b/Package.swift
@@ -37,6 +37,7 @@ let package = Package(
         ]),
         .testTarget(name: "SwagGenKitTests", dependencies: [
           "SwagGenKit",
+          "Spectre",
         ]),
         .testTarget(name: "SwaggerTests", dependencies: [
           "Swagger",

--- a/README.md
+++ b/README.md
@@ -23,7 +23,7 @@ It is an alternative the official [swagger-codegen](https://github.com/swagger-a
 `SwagGen` includes a bundled template for generating a client side Swift library for interfacing with the Swagger spec. It includes support for model inheritance, shared enums, discrete and mutable request objects, inline schemas, Codable and Equatable models, configurable options, generic networking stack, and many other niceties.
 
 ## Installing
-Make sure Xcode 10.0 is installed first.
+Make sure Xcode 10.2 is installed first.
 
 ### [Mint](https://github.com/yonaskolb/mint)
 ```sh

--- a/Sources/Swagger/SwaggerSpec.swift
+++ b/Sources/Swagger/SwaggerSpec.swift
@@ -59,7 +59,7 @@ extension SwaggerSpec {
 
     public init(string: String) throws {
         let yaml = try Yams.load(yaml: string)
-        let json = yaml as! JSONDictionary
+        let json = yaml as? JSONDictionary ?? [:]
 
         try self.init(jsonDictionary: json)
     }

--- a/Specs/Petstore/generated/Swift/Cartfile
+++ b/Specs/Petstore/generated/Swift/Cartfile
@@ -1,3 +1,3 @@
 
-github "Alamofire/Alamofire" ~> 4.8.1
+github "Alamofire/Alamofire" ~> 4.8.2
 github "antitypical/Result" ~> 4.1.0

--- a/Specs/Petstore/generated/Swift/Package.swift
+++ b/Specs/Petstore/generated/Swift/Package.swift
@@ -8,7 +8,7 @@ let package = Package(
         .library(name: "Petstore", targets: ["Petstore"])
     ],
     dependencies: [
-        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.1")),
+        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.2")),
         .package(url: "https://github.com/antitypical/Result.git", .exact("4.1.0")),
     ],
     targets: [

--- a/Specs/Petstore/generated/Swift/Petstore.podspec
+++ b/Specs/Petstore/generated/Swift/Petstore.podspec
@@ -10,6 +10,6 @@ Pod::Spec.new do |s|
     s.tvos.deployment_target = '9.0'
     s.osx.deployment_target = '10.9'
     s.source_files = 'Sources/**/*.swift'
-    s.dependency 'Alamofire', '~> 4.8.1'
+    s.dependency 'Alamofire', '~> 4.8.2'
     s.dependency 'Result', '~> 4.1.0'
 end

--- a/Specs/PetstoreTest/generated/Swift/Cartfile
+++ b/Specs/PetstoreTest/generated/Swift/Cartfile
@@ -1,3 +1,3 @@
 
-github "Alamofire/Alamofire" ~> 4.8.1
+github "Alamofire/Alamofire" ~> 4.8.2
 github "antitypical/Result" ~> 4.1.0

--- a/Specs/PetstoreTest/generated/Swift/Package.swift
+++ b/Specs/PetstoreTest/generated/Swift/Package.swift
@@ -8,7 +8,7 @@ let package = Package(
         .library(name: "PetstoreTest", targets: ["PetstoreTest"])
     ],
     dependencies: [
-        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.1")),
+        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.2")),
         .package(url: "https://github.com/antitypical/Result.git", .exact("4.1.0")),
     ],
     targets: [

--- a/Specs/PetstoreTest/generated/Swift/PetstoreTest.podspec
+++ b/Specs/PetstoreTest/generated/Swift/PetstoreTest.podspec
@@ -10,6 +10,6 @@ Pod::Spec.new do |s|
     s.tvos.deployment_target = '9.0'
     s.osx.deployment_target = '10.9'
     s.source_files = 'Sources/**/*.swift'
-    s.dependency 'Alamofire', '~> 4.8.1'
+    s.dependency 'Alamofire', '~> 4.8.2'
     s.dependency 'Result', '~> 4.1.0'
 end

--- a/Specs/Rocket/generated/Swift/Cartfile
+++ b/Specs/Rocket/generated/Swift/Cartfile
@@ -1,3 +1,3 @@
 
-github "Alamofire/Alamofire" ~> 4.8.1
+github "Alamofire/Alamofire" ~> 4.8.2
 github "antitypical/Result" ~> 4.1.0

--- a/Specs/Rocket/generated/Swift/Package.swift
+++ b/Specs/Rocket/generated/Swift/Package.swift
@@ -8,7 +8,7 @@ let package = Package(
         .library(name: "Rocket", targets: ["Rocket"])
     ],
     dependencies: [
-        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.1")),
+        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.2")),
         .package(url: "https://github.com/antitypical/Result.git", .exact("4.1.0")),
     ],
     targets: [

--- a/Specs/Rocket/generated/Swift/Rocket.podspec
+++ b/Specs/Rocket/generated/Swift/Rocket.podspec
@@ -14,6 +14,6 @@ best suits the application they are developing.
     s.tvos.deployment_target = '9.0'
     s.osx.deployment_target = '10.9'
     s.source_files = 'Sources/**/*.swift'
-    s.dependency 'Alamofire', '~> 4.8.1'
+    s.dependency 'Alamofire', '~> 4.8.2'
     s.dependency 'Result', '~> 4.1.0'
 end

--- a/Specs/TBX/generated/Swift/Cartfile
+++ b/Specs/TBX/generated/Swift/Cartfile
@@ -1,3 +1,3 @@
 
-github "Alamofire/Alamofire" ~> 4.8.1
+github "Alamofire/Alamofire" ~> 4.8.2
 github "antitypical/Result" ~> 4.1.0

--- a/Specs/TBX/generated/Swift/Package.swift
+++ b/Specs/TBX/generated/Swift/Package.swift
@@ -8,7 +8,7 @@ let package = Package(
         .library(name: "TBX", targets: ["TBX"])
     ],
     dependencies: [
-        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.1")),
+        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.2")),
         .package(url: "https://github.com/antitypical/Result.git", .exact("4.1.0")),
     ],
     targets: [

--- a/Specs/TBX/generated/Swift/TBX.podspec
+++ b/Specs/TBX/generated/Swift/TBX.podspec
@@ -10,6 +10,6 @@ Pod::Spec.new do |s|
     s.tvos.deployment_target = '9.0'
     s.osx.deployment_target = '10.9'
     s.source_files = 'Sources/**/*.swift'
-    s.dependency 'Alamofire', '~> 4.8.1'
+    s.dependency 'Alamofire', '~> 4.8.2'
     s.dependency 'Result', '~> 4.1.0'
 end

--- a/Specs/TFL/generated/Swift/Cartfile
+++ b/Specs/TFL/generated/Swift/Cartfile
@@ -1,3 +1,3 @@
 
-github "Alamofire/Alamofire" ~> 4.8.1
+github "Alamofire/Alamofire" ~> 4.8.2
 github "antitypical/Result" ~> 4.1.0

--- a/Specs/TFL/generated/Swift/Package.swift
+++ b/Specs/TFL/generated/Swift/Package.swift
@@ -8,7 +8,7 @@ let package = Package(
         .library(name: "TFL", targets: ["TFL"])
     ],
     dependencies: [
-        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.1")),
+        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.2")),
         .package(url: "https://github.com/antitypical/Result.git", .exact("4.1.0")),
     ],
     targets: [

--- a/Specs/TFL/generated/Swift/TFL.podspec
+++ b/Specs/TFL/generated/Swift/TFL.podspec
@@ -10,6 +10,6 @@ Pod::Spec.new do |s|
     s.tvos.deployment_target = '9.0'
     s.osx.deployment_target = '10.9'
     s.source_files = 'Sources/**/*.swift'
-    s.dependency 'Alamofire', '~> 4.8.1'
+    s.dependency 'Alamofire', '~> 4.8.2'
     s.dependency 'Result', '~> 4.1.0'
 end

--- a/Specs/TestSpec/generated/Swift/Cartfile
+++ b/Specs/TestSpec/generated/Swift/Cartfile
@@ -1,3 +1,3 @@
 
-github "Alamofire/Alamofire" ~> 4.8.1
+github "Alamofire/Alamofire" ~> 4.8.2
 github "antitypical/Result" ~> 4.1.0

--- a/Specs/TestSpec/generated/Swift/Package.swift
+++ b/Specs/TestSpec/generated/Swift/Package.swift
@@ -8,7 +8,7 @@ let package = Package(
         .library(name: "TestSpec", targets: ["TestSpec"])
     ],
     dependencies: [
-        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.1")),
+        .package(url: "https://github.com/Alamofire/Alamofire.git", .exact("4.8.2")),
         .package(url: "https://github.com/antitypical/Result.git", .exact("4.1.0")),
     ],
     targets: [

--- a/Specs/TestSpec/generated/Swift/TestSpec.podspec
+++ b/Specs/TestSpec/generated/Swift/TestSpec.podspec
@@ -10,6 +10,6 @@ Pod::Spec.new do |s|
     s.tvos.deployment_target = '9.0'
     s.osx.deployment_target = '10.9'
     s.source_files = 'Sources/**/*.swift'
-    s.dependency 'Alamofire', '~> 4.8.1'
+    s.dependency 'Alamofire', '~> 4.8.2'
     s.dependency 'Result', '~> 4.1.0'
 end

--- a/Templates/Swift/template.yml
+++ b/Templates/Swift/template.yml
@@ -20,7 +20,7 @@ options:
     - name: Alamofire
       pod: Alamofire
       github: Alamofire/Alamofire
-      version: 4.8.1
+      version: 4.8.2
     - name: Result
       pod: Result
       github: antitypical/Result

